### PR TITLE
[FOR REVIEW] Add ZeroEx Settler trades on BNB

### PR DESCRIPTION
**Pull request by RantumBits**
### Contribution type
Please check the type of contribution this pull request is for:

- [x] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name* zeroex_bnb.settler_trades
- **Description:** [Detailed description of the new spell(s) and their purpose] decode 0x API trades on new settler contracts on BNB chain 
- **Who are the new spell(s) for?** [Internal team or general community] both 
- **How will the new spell(s) be used downstream?** [Description of downstream usage] merged into dex-agg.trades 
- **Implementation details:** [Information on how the spell(s) are implemented]  gets taker and maker amounts only, no fills level data for intermediary swaps
- **Test instructions:** [How to test the new spell(s)] https://dune.com/queries/3921632- **Related issue(s):** [Link to related issues, if any]

### CODE REVIEW

**Comment by RantumBits** [comment]
updated to match requested changes on #6425

File: dex/models/_projects/zeroex/bnb/zeroex_bnb_schema.yml [hunk 1]
```diff
@@ -195,4 +195,56 @@ models:
       - *matcha_limit_order_flag
       - &protocol_fee_paid_eth
         name: protocol_fee_paid_eth
-        description: "The protocol fee paid in ETH"
\ No newline at end of file
+        description: "The protocol fee paid in ETH"
+  - name: zeroex_bnb_settler_trades
```
Comment (Hosuke) [inline dex/models/_projects/zeroex/bnb/zeroex_bnb_schema.yml:199]:
```suggestion
        description: "The protocol fee paid in ETH"

  - name: zeroex_bnb_settler_trades 
```

File: dex/seeds/_project/zeroex/bnb/_schema.yml [hunk 2]
```diff
@@ -56,4 +56,13 @@ seeds:
         volume_usd: double
         tx_from: varbinary
         tx_to: varbinary
-        fills_within: int
\ No newline at end of file
+        fills_within: int
+  - name: zeroex_bnb_settler_trades_sample
```
Comment (Hosuke) [inline dex/seeds/_project/zeroex/bnb/_schema.yml:60]:
```suggestion
        fills_within: int

  - name: zeroex_bnb_settler_trades_sample
```

**Comment by Hosuke** [comment]
Please resolve conflicts in other similar PRs first. Thank you.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits 

[Sample zeroex_bnb_settler_trades](https://dune.com/queries/3949381)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**